src/workerd/util/sqlite-kv.c++

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2023 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+
+#include "sqlite-kv.h"
+
+namespace workerd {
+
+SqliteKv::SqliteKv(Sqlite& db, bool): db(db) {}
+
+Sqlite& SqliteKv::ensureInitialized(Sqlite& db) {
+  // TODO(sqlite): Do this automatically at a lower layer?
```

Comment:
Non-blocking nit: this could probably use some internal protection against accentally being called multiple times in a single process even if only in a `#ifdef KJ_DEBUG` block?

Reply:
This code needs to be called once per DB, not per process. It's idempotent, so if it gets called multiple times, that's actually OK.